openstudiocore/src/pat_app/Measures/StandardReports/measure.xml

```diff
@@ -389,7 +389,6 @@
       </choices>
     </argument>
   </arguments>
-  <outputs/>
```

Comment:
@DavidGoldwasser I think this outputs element indicates that the xml had been written by OS 2.0